README.md

```diff
@@ -145,7 +145,7 @@ flutter build ipa
 ### macOS
 
 ```bash
-flutter build macos
+flutter build macoS
```

Comment:
The macOS and Linux commands must stay unchanged because they are correct already